### PR TITLE
mcos-dark-xfce-edition: init at f0ec359

### PR DESCRIPTION
**Pull request by LinArcX**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/themes/mcos-dark-xfce-edition/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "mcos-dark-xfce-edition";
```
Comment (romildo) [inline pkgs/data/themes/mcos-dark-xfce-edition/default.nix]: It is missing a version number. Maybe the date of the revision should be used for that.

Comment (romildo) [reply pkgs/data/themes/mcos-dark-xfce-edition/default.nix]: Also the base name is `McOS-MJV-Dark-XFCE-Edition`. Your name is missing the `MJV` part.

Comment (romildo) [reply pkgs/data/themes/mcos-dark-xfce-edition/default.nix]: The author has other OSX themes besides Mojave (MJV), like High Sierra (HS) and Yosemite (YS).

Comment (LinArcX) [reply pkgs/data/themes/mcos-dark-xfce-edition/default.nix]: Done!

File: pkgs/data/themes/mcos-dark-xfce-edition/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "mcos-mjv-dark-xfce-edition";
```
Comment (ryantm) [inline pkgs/data/themes/mcos-dark-xfce-edition/default.nix]:
```suggestion
  pname = "mcos-mjv-dark-xfce-edition";
```

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Closing because the opener did not address the reviewer feedback in over 1.5 years.

**Pull request closed without merging**